### ISSUE TITLE
Transactions not showing correct amount

### ISSUE THREAD

**Issue by skar4242** [TronWallet/tronwallet-play-releases#6]
OS: Windows 10
Wallet Ver: 1.1.16

When I transfer into my wallet 100 trx the transaction is shown in the wallet app as 0.0001 trx. The balance will then update with this amount. If I check tronscan with the hash I can see the correct amount of 100 trx and tronscan also shows the correct balance for my wallet.

If I reset the wallet and restore my account the balance is taken from the blockchain and all is ok. But transactions are still not being handled correctly.

I've been using the desktop app for sometime and it's been fine until now. 

TL;DR - Transfers not being handled correctly and the balance in the wallet not matching what's on the blockchain.

**Comment by pedrobertao** [comment]
We are release a fix to it soon

**Comment by pedrobertao** [comment]
Fixed on 1.1.17
